src/OpenLoco/Window.cpp

```diff
@@ -1471,4 +1471,78 @@ namespace OpenLoco::Ui
                 0x10);
         }
     }
+
+    WidgetIndex_t Window::prevAvailableWidgetInRange(WidgetIndex_t minIndex, WidgetIndex_t maxIndex)
```

Comment:
I've no idea what these functions do. The names are not great. If its just for tabs may as well say its for tabs.

Reply:
Not just the tabs, but any widget in a given range. I'm using this function for the direction and slope buttons, too. But yes, let's think of better names, then. 

It might be a useful helper function for future shortcuts as well. I thought, which is why I made it into a member function

Reply:
Would it help if we'd go with 'enabled' rather than 'available'? That is, e.g., `prevEnabledWidgetInRange` for the previous enabled widget within the provided range? Initially, I went with 'available' because it's not just checking the `enabledWidgets` mask.

Reply:
I feel like iterateAvailableWidgetsInRange and reverseIterateAvailableWidgetsInRange but then its getting too long ... bah its probably fine.

Reply:
Iterate, to me, sounds like a particular function is being executed on all elements... 😅

Reply:
Heh, yeah, iterate, to me, sounds like a particular function is being executed on all elements... Naming is hard! 😅